api/bazel/repository_locations.bzl

```diff
@@ -52,6 +55,11 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "opencensus-proto-" + OPENCENSUS_PROTO_GIT_SHA + "/src",
         urls = ["https://github.com/census-instrumentation/opencensus-proto/archive/" + OPENCENSUS_PROTO_GIT_SHA + ".tar.gz"],
     ),
+    rules_proto = dict(
```

Comment:
We're already loading `rules_proto` in https://github.com/envoyproxy/envoy/blob/3156229006a5340b65c773329070737f67e81826/bazel/repository_locations.bzl#L245. 

I think we want to do what you have in this PR, and move the repo setup to here. Can you take care of removing the `rules_proto` uses ^^ and having them reference this repository instead?

Reply:
Thanks, done.